pkg/manager/manager.go

```diff
@@ -82,34 +83,31 @@ func (k *KubeMacPoolManager) Run(rangeStart, rangeEnd net.HardwareAddr) error {
 		return fmt.Errorf("unable to create a kubernetes client error %v", err)
 	}
 
-	for k.continueToRunManager {
-		log.Info("waiting for manager to become leader")
-		err = k.waitToStartLeading()
-		if err != nil {
-			log.Error(err, "failed to wait for leader election")
-			continue
-		}
-
-		healthProbeHost, ok := os.LookupEnv("HEALTH_PROBE_HOST")
-		if !ok {
-			log.Error(err, "Failed to load HEALTH_PROBE_HOST from environment variable")
-			os.Exit(1)
-		}
+	healthProbeHost, ok := os.LookupEnv("HEALTH_PROBE_HOST")
```

Comment:
Environment variables parsing  should be on main function then pass it to `NewKubeMacPoolManager`,
then if something go wrong it could fail earlier.

We should also do something with environment variables names (like we do with constant strings - exporting them  to a file like names.go

Reply:
@RamLavi we don't need this env variables anymore since we are not using controller-runtime healtz mechanism, so we can remove the HEALT_PROBE_x variable retrieval and also manifests and remove `HealthProbeBindAddress` from manager, but this can be done at master, I am going to put a PR in place.

PR removing those variables, so we reduce the size of this PR, https://github.com/k8snetworkplumbingwg/kubemacpool/pull/184